pkgs/by-name/bo/bottles-unwrapped/package.nix

```diff
@@ -25,17 +24,18 @@
   mangohud,
   vkbasalt-cli,
   vmtouch,
+  nix-update-script,
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "bottles-unwrapped";
-  version = "51.13";
+  version = "51.15";
 
   src = fetchFromGitHub {
     owner = "bottlesdevs";
     repo = "bottles";
     rev = version;
```

Comment:
You can also switch this to `refs/tags/${version}` while you're at it. Not a blocker, but would be nice to get it out of the way.

Reply:
Took care of that.
Thanks for reviewing this.